src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -156,6 +156,8 @@ public function toolbarAction(Request $request, $token)
     /**
      * Renders the profiler search bar.
      *
+     * @param Request $request
```

Comment:
This annotation is redundant and does not add information. Please remove it.

Reply:
Done, also did as you requested in https://github.com/symfony/symfony/pull/32218#discussion_r298189660